client/state/reader/follows/actions.js

```diff
@@ -47,3 +52,52 @@ export function recordUnfollow( url ) {
 		} );
 	};
 }
+
+/**
+ * Returns an action object to signal that followed sites have been received.
+ *
+ * @param  {Array}  follows Follows received
+ * @return {Object} 		Action object
+ */
+export function receiveFollows( follows ) {
+	return {
+		type: READER_FOLLOWS_RECEIVE,
+		follows
+	};
+}
+
+/**
+ * Triggers a network request to fetch user's followed sites.
+ *
+ * @param  {Integer} page Page number of results
+ * @param  {Integer} limit Maximum number of results to return
+ * @return {Function} Action thunk
+ */
+export function requestFollows( page = 1, limit = 5 ) {
```

Comment:
That will come in a later PR. I need to port the page tracking/`isLastPage` logic from the Flux store.